docs/README.md

```diff
@@ -1,25 +1,41 @@
-# Website
+# Installing Service Workbench Documentation using local installation
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+### Prerequisites
 
-### Installation
++ Service Workbench must be installed on your local machine. For information on installing Service Workbench, read [Service Workbench Installation Guide](/docs/Service_Workbench_Installation_Guide.pdf).
```

Comment:
This step is not necessary.

Reply:
Hi Tim, what is the alternate way? Thanks